data/RI/incentives.json

```diff
@@ -858,15 +863,18 @@
     ],
     "short_description": {
       "en": "Up to $1,250 rebate for electric wiring. Available for moderate-income residents."
-    }
+    },
+    "low_income": "ri-hear-program-moderate"
   },
   {
     "id": "RI-53",
     "payment_methods": [
       "rebate"
     ],
     "items": [
-      "other"
+      "heat_pump_clothes_dryer",
+      "electric_stove",
+      "electric_stove"
```

Comment:
Don't need stove twice

Reply:
right, I guess the induction and electric stoves both map to the same key here. I'll delete